tests/ui/backtrace/synchronized-panic-handler.run.stderr

```diff
@@ -0,0 +1,5 @@
+thread '<unnamed>' panicked at $DIR/synchronized-panic-handler.rs:8:5:
+oops oh no woe is me
+note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
+thread '<unnamed>' panicked at $DIR/synchronized-panic-handler.rs:8:5:
+oops oh no woe is me
```

Comment:
do these wind up interleaved currently on some executions?

I suspect the minimum saturation for threads would have to be... larger, before seeing significant interleaving.

Reply:
this is the output from before i made this change: https://github.com/rust-lang/rust/pull/127397/commits/de14f1f932f4f11130be3fed5cd370bd0936032e
you can see it's pretty garbage. after this change i ran the same test ~25 times in a row without failures.

Reply:
oh wow.

Reply:
sorry, hadn't noticed there were 2 commits. ^^;